jib-core/src/test/java/com/google/cloud/tools/jib/image/ReproducibleLayerBuilderTest.java

```diff
@@ -106,7 +106,8 @@ public void testBuild() throws URISyntaxException, IOException {
                 .addEntry(blobA, AbsoluteUnixPath.get("/extract/here/apple/blobA"))
                 .addEntry(blobA, AbsoluteUnixPath.get("/extract/here/banana/blobA"))
                 .build()
-                .getLayerEntries());
+                .getLayerEntries(),
```

Comment:
Would be nice to have a test where we set it to something that is not the default?